libass/ass.c

```diff
@@ -602,6 +630,8 @@ static int process_styles_line(ASS_Track *track, char *str)
         track->style_format = strdup(p);
         ass_msg(track->library, MSGL_DBG2, "Style format: %s",
                track->style_format);
+        if (strcmp(p, ASS_STYLE_FORMAT) && strcmp(p, SSA_STYLE_FORMAT))
```

Comment:
Maybe it's better to have whitespace-agnostic comparison here.

Reply:
Upon closer inspection the parser doesn't care about lower-, uppercase. Case and whitespace agnostic comparison shouldn't have false positives and all false negatives would cause parser problems at another point anyway (eg `[…], Effec t, Te xt`).
It's probably better though to utilise `next_token` and `ass_strcasecmp` directly to compare the format lines.

Reply:
Done.